modules/help_editor/ajax/help.php

```diff
@@ -32,20 +32,28 @@
     include_once "helpfile.class.inc";
 
     if (!empty($moduleName)) {
-        $helpID = \LORIS\help_editor\HelpFile::hashToID(
-            md5($subpageName ?? $moduleName)
-        );
-    }
-
-    $help_file       = \LORIS\help_editor\HelpFile::factory($helpID);
-    $data            = $help_file->toArray();
-    $data['content'] = trim($data['content']);
+        try {
+            $helpID          = \LORIS\help_editor\HelpFile::hashToID(
+                md5($subpageName ?? $moduleName)
+            );
+            $help_file       = \LORIS\help_editor\HelpFile::factory($helpID);
+            $data            = $help_file->toArray();
+            $data['content'] = trim($data['content']);
 
-    if (empty($data['updated'])) {
-        $data['updated'] = "-";
-        // if document was never updated should display date created
-        if (!empty($data['created'])) {
-            $data['updated'] = $data['created'];
+            if (empty($data['updated'])) {
+                $data['updated'] = "-";
+                // if document was never updated should display date created
+                if (!empty($data['created'])) {
+                    $data['updated'] = $data['created'];
+                }
+            }
+        } catch (\NotFound $e) {
```

Comment:
I'm unsure having a giant try catch here is best compared to encompassing just the lines that throw the error and then handling it from there.

Reply:
ill try make this change